corehq/apps/hqadmin/views/users.py

```diff
@@ -187,11 +187,14 @@ def _get_restore_response(self):
     @staticmethod
     def _parse_reports(xpath, xml_payload):
         reports = xml_payload.findall(xpath)
-        report_row_counts = {
-            report.attrib['report_id']: len(report.findall('{{{0}}}rows/{{{0}}}row'.format(RESPONSE_XMLNS)))
-            for report in reports
-            if 'report_id' in report.attrib
-        }
+        report_row_counts = {}
+        for report in reports:
+            if 'report_id' in report.attrib:
+                report_id = report.attrib['report_id']
+                if 'id' in report.attrib:
+                    report_id = '--'.join([report.attrib['id'], report_id])
```

Comment:
nit: you can use tuples as dict keys as an alternative to this join

Reply:
@esoergel 
Just wanted it to be easier to read.
I believe tuple would be
`('33955627e1e148dba55b373d7150378b', '339452627e1e148dba55b373d7150378b')`
and this would be
`33955627e1e148dba55b373d7150378b--339452627e1e148dba55b373d7150378b`

Reply:
yeah, makes sense - forgot it was being displayed to the user when I left that comment